app/models/concerns/CasaCase/validations.rb

```diff
@@ -7,11 +7,13 @@ module CasaCase::Validations
     validates :birth_month_year_youth, presence: true
     validates :birth_month_year_youth, comparison: {
       less_than_or_equal_to: -> { Time.now.end_of_day },
-      message: "is not valid: Youth's Birth Month & Year cannot be a future date."
+      message: "is not valid: Youth's Birth Month & Year cannot be a future date.",
```

Comment:
So this will output a pretty verbose message like the ones removed in specs below. Instead, we could do:
```suggestion
      message: "cannot be a future date.",
```
And in `config/locales/en.yml`:
```yml
en:
  activerecord:
    attributes:
      casa_case:
        birth_month_year_youth: Youth's Birthday
```
(Or the original "Youth's Birth Month & Year")

ActiveRecord looks for that automatically to output "Youth's Birthday cannot be a future date." Much easier on the user than ""Birth month year youth is not valid: Youth's Birth Month & Year cannot be a future date."

Also note that this message in particular already has a translation available, so this message could even be 
```suggestion
      message: :cant_be_future,
```

Reply:
And if you want to test the messages in `spec/models/casa_case_spec.rb`, that would be super.

Reply:
I agree with you. I will work on it. :)